fee_calculator/apps/calculator/fixtures/price.json

```diff
@@ -1357904,5 +1357976,1445 @@
     "strict_range": false,
     "modifiers": []
   }
+},
+{
+  "model": "calculator.price",
+  "pk": 102357,
+  "fields": {
+    "scenario": 8,
+    "scheme": 3,
+    "advocate_type": "JUNIOR",
+    "offence_class": null,
+    "fee_type": 13,
+    "unit": "HOUR",
+    "fixed_fee": "0.00000",
+    "fee_per_unit": "40.00000",
+    "limit_from": 9,
```

Comment:
From looking at the [applicable regs](http://www.legislation.gov.uk/uksi/2013/435/schedule/1/paragraph/19/made), and without fully understanding the code itself, are the `limit_from` and `limit_too` representing the following:  7 to 8 (1 extra CV), 9 to 12 (two extra), 13 to 18 (three extra)?

Reply:
as discussed 👍